frontend/src/app/commonComponents/Button.jsx

```diff
@@ -17,12 +17,14 @@ const Button = ({
   secondaryDisabled,
   label,
   outline,
+  unstyled = false,
   addClass,
 }) => {
   let classNames = classnames(
     {
       "usa-button": true,
       "usa-button-big": big,
+      "usa-button--unstyled": unstyled,
```

Comment:
This whole approach seems like it would eventually blow up, but hey, I'm just a Java dinosaur... 🦕

Reply:
Yeah, the Trussworks component has a `variant` prop where you pass in one of these, since they're mutually exclusive. I decide to continue the insane pattern. When in Rome...